pallets/parachain-staking/src/lib.rs

```diff
@@ -440,25 +440,8 @@ pub mod pallet {
 		}
 
 		fn on_runtime_upgrade() -> Weight {
-			log::info!(target: "parachain-staking", "Initializing parachain-staking storage with config defaults");
-
-			// Set collator commission to default config
-			<CollatorCommission<T>>::put(T::DefaultCollatorCommission::get());
-			// Set parachain bond config to default config
-			<ParachainBondInfo<T>>::put(ParachainBondConfig {
-				// must be set soon; if not => due inflation will be sent to collators/delegators
-				account: T::AccountId::decode(&mut sp_runtime::traits::TrailingZeroInput::zeroes())
-					.expect("infinite length input; no invalid inputs for type; qed"),
-				percent: T::DefaultParachainBondReservePercent::get(),
-			});
-			// Set total selected candidates to minimum config
-			<TotalSelected<T>>::put(T::MinSelectedCandidates::get());
-			// Start Round 1
-			let round: RoundInfo<T::BlockNumber> =
-				RoundInfo::new(1u32, 0u32.into(), T::DefaultBlocksPerRound::get());
-			<Round<T>>::put(round);
-
-			T::DbWeight::get().writes(4)
+			use frame_support::traits::OnRuntimeUpgrade;
+			crate::migrations::SplitDelegatorStateIntoDelegationScheduledRequests::<T>::on_runtime_upgrade()
```

Comment:
In practice, this should barely affect us at this moment since we've just recently added parachain-staking and have no community delegators (atm). We should still turn the staking valve off during the upgrade.